src/components/main-view/main-view.jsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import axios from "axios";
+
+import { MovieCard } from "../movie-card/movie-card";
+import { MovieView } from "../movie-view/movie-view";
+
+export class MainView extends React.Component {
+  constructor() {
+    // Call the superclass constructor
+    // so React can initialize it
+    super();
+
+    this.state = {
+      movies: null,
+      selectedMovie: null,
+    };
+  }
+
+  // One of the "hooks" available in a React Component
+  componentDidMount() {
+    axios
+      .get("https://myflix89.herokuapp.com/movies")
+      .then((response) => {
+        // Assign the result to the state
+        this.setState({
+          movies: response.data,
+        });
+      })
+      .catch(function (error) {
+        console.log(error);
+      });
+  }
+
+  onMovieClick(movie) {
+    this.setState({
+      selectedMovie: movie,
+    });
+  }
+
+  render() {
+    const { movies, selectedMovie } = this.state;
+
+    // Before the movies have been loaded
+    if (!movies) return <div className="main-view" />;
+
+    return (
+      <div className="main-view">
+        {selectedMovie ? (
+          <MovieView movie={selectedMovie} />
```

Comment:
You can pass an `onClick` function to MovieView as you did with `MovieCard`. Instead of passing a movie in the parameter you can pass `null` so this will change the state back to what it was when you first hit the movies page.

i.e 
```
 <MovieView movie={selectedMovie} onClick={() => this.onMovieClick(null)}
```